README.md

```diff
@@ -155,6 +155,22 @@ user/group.
 You can restrict the allowed mounts points for the pod's volumes. By default mounts under "/var" are
 allowed, but this can be changed via the `--dir` or `-d` flag.
 
+### Running Without Root Permissions
+
+Some work has been done to be able to run `systemk` without root permissions. This means a couple of
+things:
+
+1. Packages can't be installed, so you'll need to use a path as the image name (See "Binary Exists in
```

Comment:
You can use [markdown anchors to link to an existing section](https://gist.github.com/asabaylus/3071099).

Reply:
yeah, problem is that only works when viewing html...

Reply:
That's how most people read it, you terminal-fanatic hermit :-p

Reply:
But fine.